addons/payment_adyen/models/payment_transaction.py

```diff
@@ -282,3 +283,84 @@ def _adyen_tokenize_from_feedback_data(self, data):
         _logger.info(
             "created token with id %s for partner with id %s", token.id, self.partner_id.id
         )
+
+    def _adyen_extend_api_data(self, data):
```

Comment:
Please move this method and the two others into a new `payment_adyen/utils.py` file.

Reply:
Why? I need to use `self`

```
order = self.sale_order_ids[:1]
```

Reply:
Because it's much cleaner and because it's not only used from the models. You can pass the transaction as a parameter.